curvefs/src/metaserver/inode_storage.h

```diff
@@ -43,6 +43,7 @@
 #include "curvefs/src/metaserver/storage/utils.h"
 #include "src/common/concurrent/rw_lock.h"
 
+#define DELETING_PREFIX "deleting_"
```

Comment:
The raw string prefix is inappropriate, you can add a new key type in https://github.com/opencurve/curve/blob/dd8ce61654940a8e789960ce5c03c1c84b09bf0a/curvefs/src/metaserver/storage/converter.h#L40